@here/harp-mapview/lib/composing/MSAARenderPass.ts

```diff
@@ -178,18 +178,19 @@ export class MSAARenderPass extends Pass {
             const sampleWeight = 1.0 / offsets.length + uniformCenteredDistribution / 32;
 
             this.m_quadUniforms.opacity.value = sampleWeight;
-            renderer.render(scene, camera, this.m_renderTarget, true);
+
+            const oldRenderTarget = renderer.getRenderTarget();
+            renderer.setRenderTarget(this.m_renderTarget);
+            renderer.clear();
+            renderer.render(scene, camera);
 
             // 6. Render the quad on top of the previous renders.
+            renderer.setRenderTarget(this.renderToScreen ? undefined : writeBuffer);
             if (i === 0) {
                 renderer.setClearColor(0x000000);
+                renderer.clear();
             }
-            renderer.render(
-                this.m_quadScene,
-                this.m_localCamera,
-                this.renderToScreen ? undefined : writeBuffer,
-                i === 0
-            );
+            renderer.render(this.m_quadScene, this.m_localCamera);
```

Comment:
The performance / antialias example does not work anymore when using this internal MSAA, it breaks in the TryMe. Your code seems ok, so I tried to download it but got an error at downloading, so cannot check immediately,

Reply:
Thanks, will check this.

Reply:
Apparently, 3JS has bug as it requires specifies:
```setRenderTarget(v: WEBGLRendereTarget | undefined)```

But throws exception, when you pass `undefined`.
`null` works as it was always working.

See https://github.com/heremaps/harp.gl/pull/286/commits/f8f96cc850e444e8eb93a2f18cb2d5658e556b9f#diff-8fbcf5dde65d97a3aed881c2d420db2cR143